### DISCUSSION TITLE
Live accuracy heatmap?

### DISCUSSION THREAD

**Discussion by Sclafus** [ppy/osu#25412] [Ideas]
Hello everyone,
Would it be possible to have the accuracy heatmap live on screen? I think it could really help when trying to understand if you are over-aiming or under-aiming during gameplay.
![image](https://github.com/ppy/osu/assets/31241607/30c8ec9b-31e6-42b6-8d0b-a9464a4240a8)

**Comment by nagi-desuuu** [comment]
If this were to become a real thing, it would be an optional component accessible via the skin editor.
Just saying...

But given the ranked project (and preparations for potential release), feature requests are a bit of a non-priority right now.

**Reply by nagi-desuuu** [reply]
But they might consider. Who knows?

**Reply by Sclafus** [reply]
> If this were to become a real thing, it would be an optional component accessible via the skin editor. Just saying...

Agreed, that's indeed the way of implementing this

I know it would be low priority, it's just an idea anyway :D

**Reply by cdwcgt** [reply]
this is interesting for me.
Maybe I will have a try.

**Reply by Gabixel** [reply]
I will hazard to say I think this is covered in discussion #13048, although I think it has been somewhat forgotten since almost all the statistics in the OP are already in the game (UR and CPS), but some still seem interesting (e.g. the number of wrong clicks, or the cursor speed).
I also approve that some statistics visible in the replay details can be interesting to see during gameplay (especially for spectators).
(I also only just found that discussion.)

> Bullseye / scatter plot of cursor position from circle center

**Comment by peppy** [comment]
I'd like to see this.

**Comment by cdwcgt** [comment]
https://github.com/ppy/osu/assets/34775378/4b4cbf4f-4b14-4499-9dcb-c65d5a441361

Mostly refer to `BarHitErrorMeter`

maybe not the same thing

**Reply by Sclafus** [reply]
This is indeed what I meant!

**Reply by Sclafus** [reply]
#25716

**Reply by huyenden** [reply]
so great :+1:
